SwiftI18n/Classes/Views/BaseViews/ViewController+I18n_loc.swift

```diff
@@ -22,5 +24,27 @@ public extension UIViewController {
             loc_localeDidChange()
         }
     }
-    
+
+    @IBInspectable var locAccessibilityLabelKey: String? {
+        get {
+            return loc_keysDictionary[UIViewController.loc_accessibilityLabelKey]
+        }
+
+        set(newValue) {
+            loc_keysDictionary[UIViewController.loc_accessibilityLabelKey] = newValue
+            loc_localeDidChange()
+        }
+    }
+
+    @IBInspectable var locAccessibilityHintKey: String? {
+        get {
+            return loc_keysDictionary[UIViewController.loc_accessibilityHintKey]
+        }
+
+        set(newValue) {
+            loc_keysDictionary[UIViewController.loc_accessibilityHintKey] = newValue
+            loc_localeDidChange()
```

Comment:
I can see that `loc_localeDidChange` is called in all places (all new codes you added). 
Do we have to call it everywhere? E.g., is it possible that someone will just use Hint without Label Key, or even use anything related to a11ly without using the main TitleKey? 

cc @Truba (tagging you for the sanity check regarding the multiple `loc_localceDidChange` calls)